fairer-iOS/fairer-iOS/Screens/HouseMakeName/HouseMakeNameViewController.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  HouseMakeNameViewController.swift
+//  fairer-iOS
+//
+//  Created by 김유나 on 2022/09/29.
+//
+
+import UIKit
+
+import SnapKit
+
+final class HouseMakeNameViewController: BaseViewController {
+
+    // MARK: - property
+
+    private let backButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.frame = CGRect(x: 0, y: 0, width: 44, height: 44)
+        button.setImage(ImageLiterals.navigationBarBackButton, for: .normal)
+        button.tintColor = .gray800
+        return button
+    }()
+    private let writeNamePrimaryLabel: UILabel = {
+        let label = UILabel()
+        label.setTextWithLineHeight(text: TextLiteral.houseMakeNameViewControllerWriteNamePrimaryLabel, lineHeight: 28)
+        label.font = .h2
+        label.textColor = .gray800
+        return label
+    }()
+    private let writeNameSecondaryLabel: UILabel = {
+       let label = UILabel()
```

Comment:
p5;

요기 한칸만 띄어주십셔... ^______^
```suggestion
        let label = UILabel()
```

Reply:
[bd55690](https://github.com/fairer-iOS/fairer-iOS/pull/33/commits/bd556906cc80365852981d7e1803444ac8ed881c)
수정해서 반영했습니다 !